packages/editor/src/components/common/topbar/help/index.tsx

```diff
@@ -56,8 +56,8 @@ const HelpDropdownDialog = ({openAboutModal}: IProps) => (
                 </a>
             </li>
             <li>
-                <a className={style.container} href='https://gitter.im/ethereum/documentation' target='_blank' rel='noopener noreferrer' title="Ethereum' community">
-                    Join our Community!
+                <a className={style.container} href='https://gitter.im/ethereum/documentation' target='_blank' rel='noopener noreferrer'>
```

Comment:
@Spejbl Why removing the title?

Reply:
@javier-tarazaga It seemed like a duplication since it already says "Eth docs" in the name of the button. Also it was only link with title inside help menu. Should I revert it?